src/miso-soup.md

```diff
@@ -0,0 +1,26 @@
+# Miso-soup
+
+Miso-soup is Japanese traditional home cooking.
+
+## Ingredients
+
+- dashi stock
+- miso paste
+- tofu(optional)
+- seaweed(optional)
+- green onion(optional)
+
+## Directions
+
+1. put about half of the water in the pot and bring the water to a boil.
```

Comment:
Half of what amount? Is there a specific amount of water needed for this recipe?